internal/controller/supervisorconfig/activedirectoryupstreamwatcher/active_directory_upstream_watcher.go

```diff
@@ -226,7 +226,7 @@ type UpstreamActiveDirectoryIdentityProviderICache interface {
 
 type activeDirectoryWatcherController struct {
 	cache                                   UpstreamActiveDirectoryIdentityProviderICache
-	validatedSecretVersionsCache            upstreamwatchers.SecretVersionCacheI
+	validatedSecretVersionsCache            upstreamwatchers.ValidatedSettingsCacheI
```

Comment:
Maybe change this variable name to `validatedSettingsCache` as well.

Reply:
Good catch! I'll change them all across all files. Not sure how I missed that. 😳